### PR TITLE
20-install-mui

### LINKED ISSUE #20: Install MUI to frontend

### PR DESCRIPTION
**Pull request by DLittlefield81**
added MUI with emotion.
to test added styled button to home page with style file and folder.

### CODE REVIEW

File: src/styles/styles.js [hunk 1]
```diff
@@ -0,0 +1,67 @@
+import { css, Global, keyframes } from '@emotion/react'
```
Comment (rmc3408) [inline src/styles/styles.js]: must be TS file, no JS.

Comment (rmc3408) [reply src/styles/styles.js]: In future try better fileName. animation.ts

File: src/app/page.tsx [hunk 2]
```diff
@@ -1,12 +1,13 @@
 "use client";
 
-
+import { Animated, Basic, bounce, Combined } from '../styles/styles'
```
Comment (rmc3408) [inline src/app/page.tsx]: Are you using Combined, Basic ??? if not, remove it.  Also remove from the `style.js` file

**Review by rmc3408 — COMMENTED** [review]
You need to change PR naming and solve some git conflicts.

File: src/styles/styles.js (hunk 1 above)
Comment (DLittlefield81) [reply src/styles/styles.js]: updated filename

File: src/app/page.tsx (hunk 2 above)
Comment (DLittlefield81) [reply src/app/page.tsx]: removed extra variables

**Review by rmc3408 — APPROVED** [review]
perfect

**Pull request closed without merging**